src/libs/sys-util-rs/src/cpu.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2019-2021 Alibaba Cloud
+// Copyright (c) 2019-2021 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Test whether two CPU sets are equal.
+pub fn is_cpu_list_equal(left: &[u32], right: &[u32]) -> bool {
```

Comment:
Would it be worth making "cpu list" an actual type.   That way you can `impl Eq` instead of having to explicitly call this function.  It also makes it easier to change the implementation (say to a `HashSet` or a bitmap) if that turns out to be worthwhile.

Reply:
done

Reply:
It's really a great suggestion, result in much more better abstraction.